### PR TITLE
Vattenfall InCharge documentation

### PR DESCRIPTION
**Pull request by tgriek**
## Proposed change
Documentation of the Vattenfall InCharge integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- [Link ](https://github.com/home-assistant/core/pull/89548 )to parent pull request in the codebase
- [Link](https://github.com/home-assistant/brands/pull/4206) to parent pull request in the Brands repository

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/incharge.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ha_codeowners:
+  - '@tgriek'
+ha_domain: incharge
+ha_platforms:
+  - sensor
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+This is a sensor to collect information from your charging station managed by Vattenfall InCharge using [Vattenfall InCharge](https://myincharge.vattenfall.com/).
```
Comment (frenck) [inline source/_integrations/incharge.markdown]:
```suggestion
This integration provides sensors to collect information from your charging station managed by Vattenfall InCharge using [Vattenfall InCharge](https://myincharge.vattenfall.com/).

```

File: source/_integrations/incharge.markdown [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: Vattenfall InCharge
+description: Instructions on how to integrate your Vattenfall InCharge managed charging stations within Home Assistant.
+ha_category:
+  - Energy
+  - Sensor
+ha_release: 2023.3.4
```
Comment (frenck) [inline source/_integrations/incharge.markdown]: New features can never be shipped in patch releases 😉 

```suggestion
ha_release: 2023.4.0

```

File: source/_integrations/incharge.markdown [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  - Sensor
+ha_release: 2023.3.4
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@tgriek'
+ha_domain: incharge
+ha_platforms:
+  - sensor
+ha_config_flow: true
+ha_integration_type: integration
```
Comment (frenck) [inline source/_integrations/incharge.markdown]: I think this is a service?

```suggestion
ha_integration_type: service

```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/incharge.markdown (hunk 3 above)
Comment (tgriek) [reply source/_integrations/incharge.markdown]: Not sure? @klaasnicolaas ?

Comment (frenck) [reply source/_integrations/incharge.markdown]: No need to ping Klaas for that, see: <https://developers.home-assistant.io/blog/2022/10/24/hubs-devices-services-virtual-integrations>

(noticed it was also missing form the manifest in the core PR).

Comment (tgriek) [reply source/_integrations/incharge.markdown]: Klaas added a new-integration label that confused me. In any case I am fine with whatever suits the standards guys.

Comment (frenck) [reply source/_integrations/incharge.markdown]: > Klaas added a new-integration label that confused me.

It is new integration 🤷 
This defines the type of integration it is.

> whatever suits the standards guys.

It is not about standards.

Please note: What Klaas added are GitHub labels and have nothing to do with the contents/code itself.

Comment (tgriek) [reply source/_integrations/incharge.markdown]: Gotcha. The idea and functioning behind this new integration is very similar to:
[Wallbox](https://www.home-assistant.io/integrations/wallbox/)
[Growatt](https://www.home-assistant.io/integrations/growatt_server/)
What makes those of the type integration and this one a service?

Comment (frenck) [reply source/_integrations/incharge.markdown]: Those are existing integrations that haven't specified them yet (please note how the linked blog post is just a couple of months old). For new integrations, it should just be set correctly from the get-go.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**